package.json

```diff
@@ -51,9 +56,11 @@
     "stripe": "^16.1.0",
     "swagger-model-validator": "^3.0.21",
     "swagger-ui-express": "^5.0.0",
+    "typedoc": "^0.26.7",
```

Comment:
It seems we only use `typedoc` in development environments and never in production environments, so maybe move it to the dev dependencies?